index.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="styles.css">
+    <title>Survey_form</title>
+</head>
+<body>
+    <h2 id="title">Microverse survey Form</h2>
```

Comment:
consider the semantics of h-tags. Therefore use h1 tag here.
1. H1-Head
     1.1 H2-Head
         1.1.1 H3-Head
         1.1.2 H3-Head
         ...
     1.2 H2-Head
         1.2.1 H3-Head
         ...
     1.3 H2-Head
     ...

Reply:
- [ ] Hi @henry-dura, Good job for using [GitHub inline comments feature](https://docs.github.com/en/github/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/reviewing-proposed-changes-in-a-pull-request) and provide for reference for suggested changes, **As requested by the previous reviewer**, Kindly makes sure that you phrased the comments politely, as suggestions instead of orders and includes an explanation, a "why", for each requested change.
  
  Ex. "I think it would be better to use h1 tags for the .first-title element instead of an h2 tag, that would improve the HTML structure and make the page more accessible"